### PR TITLE
[Fix #12] Fix and change md files

### LINKED ISSUE #12: README revision
I think we should double check readme before releasing and adds some useful links and information.  

E.g.  link to rubocop doc and etc.

### PR DESCRIPTION
**Pull request by roman-dubrovsky**
Close #12

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,24 +4,28 @@ The format is described in [Contributing notes](CONTRIBUTING.md#changelog-entry-
...
 
-## 0.1.0 (date here e.g 2019-05-25)
 
-### Added
+### Changed
 
-* Base config with community rules and some basic override rules.
+* The limit for size of line was changed to 120 symbols ([@r.dubrovsky][])
+* Disable `Metrics/BlockLength` cop for rspec files ([@r.dubrovsky][])
+* Exclude rubocop checking for some config diractories ([@r.dubrovsky][])
```
Comment (AleksSenkou) [inline CHANGELOG.md]: `directories`

File: CHANGELOG.md [hunk 2]
```diff
@@ -4,24 +4,28 @@ The format is described in [Contributing notes](CONTRIBUTING.md#changelog-entry-
...
+* Enable `Bundler/DuplicatedGem` cop by default ([@r.dubrovsky][])
+* Enable `Bundler/OrderedGems` cop with allowing ordering by groups ([@r.dubrovsky][])
 
-## 0.1.0 (date here e.g 2019-05-25)
 
-### Added
+### Changed
 
-* Base config with community rules and some basic override rules.
+* The limit for size of line was changed to 120 symbols ([@r.dubrovsky][])
```
Comment (AleksSenkou) [inline CHANGELOG.md]: For most records `present simple` is used. Can we go with it here too?

File: CHANGELOG.md [hunk 3]
```diff
@@ -4,24 +4,28 @@ The format is described in [Contributing notes](CONTRIBUTING.md#changelog-entry-
...
+* Exclude rubocop checking for some config diractories ([@r.dubrovsky][])
+* Enable prafering double quotes for string literals ([@r.dubrovsky][])
+* Do not add spaces beetwen hash literal braces (cop `Layout/SpaceInsideHashLiteralBraces`) ([@r.dubrovsky][])
+* Prefer normal style for `Layout/IndentationConsistency` cop for Rails apps too ([@r.dubrovsky][])
+* Change style to `variable` for `Layout/EndAlignment` cop ([@r.dubrovsky][])
+* Change style to `with_fixed_indentation` with indentation width 2 for `Layout/AlignParameter` cop ([@r.dubrovsky][])
+* Always ingone hash aligning for key word arguments (cop: `Layout/AlignHash`) ([@r.dubrovsky][])
+
+[@r.dubrovsky]: https://github.com/roman-dubrovsky
+[@aleks]: https://github.com/AleksSenkou
```
Comment (AleksSenkou) [inline CHANGELOG.md]: @roman-dubrovsky thoughts about marking those who helped with code review?

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -27,7 +27,7 @@ do so.
 * If your change has a corresponding open GitHub issue, prefix the commit message with `[Fix #github-issue-number]`.
 * Make sure to add tests for it. This is important so we don't break it
   in a future version unintentionally.
-* If you add new code-style rule or cop, add some words about it in the [Code Style Notes](CHANGEME.md).
+* If you add new code-style rule or cop, add some words about it in the [Code Style Notes](STYLE_GUIDE.md).
```
Comment (AleksSenkou) [inline CONTRIBUTING.md]: ❤️

File: CHANGELOG.md (hunk 3 above)
Comment (roman-dubrovsky) [reply CHANGELOG.md]: @AleksSenkou have you meant doing it for the current changes or in general?

Comment (AleksSenkou) [reply CHANGELOG.md]: @roman-dubrovsky not sure, just wondering

Comment (roman-dubrovsky) [reply CHANGELOG.md]: @AleksSenkou no thoughts. In general, I don't think that it makes sense. Usually, we'll make the big impact of some feature (e.g. describe and implement some feature). But if the feature creator feels that the reviewer made a great impact he can add this person to the contributes as well. Does it make sense?

Comment (AleksSenkou) [reply CHANGELOG.md]: @roman-dubrovsky yeah, sounds reasonable

**Pull request merged**